### DISCUSSION TITLE
W instead of kW

### DISCUSSION THREAD

**Discussion by ChrisKoh83** [ulic75/power-flow-card#92] [Q&A] [answered]
Is where a possibility to switch from kW to W?

my sensors are reading the Power values in W. 

Thanks

**Comment by ulic75** [comment, marked as answer]
You can use the `watt_threshold` option listed in the readme to configure there.  Setting it to 0 will always show kilowatts, setting it to very high value will essentially always show watts.  Setting it somewhere else allows it to change from kilowatts to watts when under the threshold.
